tests/ci-run-e2e.sh

```diff
@@ -3,14 +3,10 @@
 set -xe
 
 if [[ $# -ne 2 ]]; then
-	echo "TEST_CASE TARGET_DRIVER_VERSION are required"
+	echo "TEST_CASE TEST_CASE_ARGS are required"
```

Comment:
Similar to a previous comment, can we rename the second argument to `OPERATOR_OPTIONS` /  `GPU_OPERATOR_OPTIONS`.

Reply:
I made it generic, so we can use the same script to run other tests like vectoradd, network operator, etc.  This one will be used only for target testing scripts.
The exec script (upgrade script) can be used for system script execution or non-deployment scripts separately.

Reply:
Okay sure, that is fair.